src/langcheck/metrics/eval_clients/_openai.py

```diff
@@ -121,7 +122,7 @@ def get_text_responses(
             A list of responses to the prompts. The responses can be None if the
             evaluation fails.
         """
-        config = {"model": "gpt-3.5-turbo"}
+        config = {"model": "gpt-35-turbo"}
```

Comment:
I think `gpt-3.5-turbo` is the valid request parameter

https://platform.openai.com/docs/models/gpt-3-5-turbo

Reply:
Ah I forgot to revert this, thanks!